src/transport/raw/Base.h

```diff
@@ -77,11 +95,26 @@ class Base
      */
     virtual bool CanListenMulticast() { return false; }
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+    /**
+     * Connect to the specified peer.
+     */
+    virtual CHIP_ERROR ConnectToPeer(const PeerAddress & address, Transport::AppConnCallbackCtxt * appState)
```

Comment:
Should mention TCP in naming, if it's TCP-specific.

Reply:
There was a Disconnect() API for TCP that was pre-existing, and this was aligned with that. TCP was expected to be implied here. Do you prefer it be TCPConnectToPeer() and TCPDisconnect() or something like that?

Reply:
Renamed the APIs around connecting/disconnecting to TCPConnect()/TCPDisconnect().